tests/morpheus_llm/services/test_kinetica_vector_db_service.py

```diff
@@ -0,0 +1,322 @@
+#!/usr/bin/env python
+# SPDX-FileCopyrightText: Copyright (c) 2023-2024, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import random
+
+import numpy as np
+import pytest
+import cudf
+
+from morpheus_llm.service.vdb.kinetica_vector_db_service import KineticaVectorDBService
+
+
+
+@pytest.mark.kinetica
+def test_has_store_object(kinetica_service: KineticaVectorDBService):
+    # Check if a non-existing collection exists in the Kinetica server.
+    collection_name = "non_existing_collection"
+    assert not kinetica_service.has_store_object(collection_name)
+
+
+@pytest.mark.kinetica
+def test_create_and_drop_collection(kinetica_type: list[list], kinetica_service: KineticaVectorDBService):
+    collection_name = "test_collection"
+
+    # Make sure to drop any existing collection from previous runs.
+    kinetica_service.drop(collection_name)
+
+    # Create a collection and check if it exists.
+    kinetica_service.create(collection_name, kinetica_type)
```

Comment:
The `kinetica_type` needs to be a keyword argument here, as the second positional argument for this method is `overwrite`.
```suggestion
    kinetica_service.create(collection_name, table_type=kinetica_type)
```

Reply:
Rectified this.